lib/fluent/plugin/out_file.rb

```diff
@@ -23,9 +23,13 @@ class FileOutput < TimeSlicedOutput
       'gzip' => :gz,
     }
 
+    desc "Specify filepath to output."
     config_param :path, :string
+    desc "Choose output format. You can choose it from registered formatter plugins."
     config_param :format, :string, :default => 'out_file'
+    desc "Disable path increment."
     config_param :append, :bool, :default => false
+    desc "Compress chunks with specified compression algorithm."
```

Comment:
Document says "Compresses flushed files using gzip.".

Should I use this one or your suggestion?